src/main/java/io/temporal/samples/dsl/SampleActivities.java

```diff
@@ -0,0 +1,77 @@
+package io.temporal.samples.dsl;
+
+import io.temporal.activity.Activity;
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+
+public class SampleActivities {
+  @ActivityInterface
+  public interface SampleActivities1 {
+    @ActivityMethod
+    String getInfo1();
```

Comment:
Is it possible to have the same method names with different Activity interfaces?  I found that if i try to registry multiple activity interfaces i can't use the same method or it conflicts